src/widgets/call-panel.jsx

```diff
@@ -308,6 +334,13 @@ class CallPanel extends React.PureComponent {
   handleTrackEvent(event) {
     // Remote video becomes available.
     this.remoteRef.current.srcObject = event.streams[0];
+
+    if (event.track.kind == 'video') {
+      // Redraw the screen when remote video stream state changes.
+      event.track.onended = _ => { this.forceUpdate(); };
+      event.track.onmute = () => { this.forceUpdate(); };
```

Comment:
Consistently `_` or `()` ?

Just `event.track.onended = _ => this.forceUpdate();`

Reply:
Done.